### ISSUE TITLE
nomad volume status -verbose command crashes

### ISSUE THREAD

**Issue by ivoronin** [hashicorp/nomad#10816]
`nomad volume status -verbose` command crashes because of out of range index access:

```sh
~# nomad volume status -verbose
Container Storage Interface
ID                Name              Plugin ID  Schedulable  Access Mode
volume1           volume1           gcepd      true         single-node-writer
volume2           volume2           gcepd      true         single-node-writer
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/hashicorp/nomad/command.(*VolumeStatusCommand).listVolumes(0xc000117110, 0xc000262ee0, 0x70)
        github.com/hashicorp/nomad/command/volume_status_csi.go:123 +0x671
github.com/hashicorp/nomad/command.(*VolumeStatusCommand).csiStatus(0xc000117110, 0xc000262ee0, 0x0, 0x0, 0x0)
        github.com/hashicorp/nomad/command/volume_status_csi.go:23 +0x5fc
github.com/hashicorp/nomad/command.(*VolumeStatusCommand).Run(0xc000117110, 0xc000132030, 0x1, 0x1, 0xc0001b1ed0)
        github.com/hashicorp/nomad/command/volume_status.go:126 +0x46a
github.com/mitchellh/cli.(*CLI).Run(0xc0001da3c0, 0xc0001da3c0, 0xc00000f080, 0x37)
        github.com/mitchellh/cli@v1.1.0/cli.go:260 +0x41a
main.RunCustom(0xc000132010, 0x3, 0x3, 0xc000102058)
        github.com/hashicorp/nomad/main.go:142 +0x4a7
main.Run(...)
        github.com/hashicorp/nomad/main.go:87
main.main()
        github.com/hashicorp/nomad/main.go:83 +0x65
```

Something is definitely wrong with this code:

https://github.com/hashicorp/nomad/blob/09bd33640bd6732d1b1b33d2229b744157e959fe/command/volume_status_csi.go#L122-L123

**Comment by tgross** [comment]
Hi @ivoronin! Can you provide the output of `nomad version` please (from the CLI and the server would be helpful).

**Comment by ivoronin** [comment]
Both clients and servers are running 1.1.1:
```
# nomad version
Nomad v1.1.1 (7feec97c04de4f8afff54ca9e56d66a61dfbfeb3)
```

**Comment by tgross** [comment]
Thanks. Yeah that's obviously a bug. 🤦 Looks like it first landed in 1.1.0. I'll push a PR up to fix it and it'll go out in the next patch.

**Comment by tgross** [comment]
#10818 has been merged and will land in Nomad 1.1.3.
